components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -2026,6 +2026,48 @@ paths:
           schema:
             $ref: '#/definitions/Error'
 
+  ######################################################
+  # AWS Lambda
+  ######################################################
+  /apis/{apiId}/amznResourceNames:
+    #-----------------------------------------------------
+    # Retrieve the ARNs of AWS Lambda functions
+    #-----------------------------------------------------
+    get:
+      security:
+        - OAuth2Security:
+            - apim:api_view
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" https://127.0.0.1:9443/api/am/publisher/v1.0/apis/64eca60b-2e55-4c38-8603-e9e6bad7d809/amznResourceNames"
+      x-wso2-request: |
```

Comment:
let's add x-wso2-example where move sample requests and responses to an example file